### PR TITLE
Added title and language options

### PR DESCRIPTION
**Pull request by VJ1224**

### CODE REVIEW

File: start-new-game.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
-I'm normal, so [I choose you](english/normal-dungeon/0/0.md)
+# Welcome to the Dungeon!
 
-[不到長城非好漢](chinese/great-wall/0/0.md)
\ No newline at end of file
+Choose your language:
```
Comment (ming-tsai) [inline start-new-game.md:3]: This will be a list of the dungeon, so it is not the language list.
It could choose the dungeon.

Comment (VJ1224) [reply start-new-game.md:3]: Alright, maybe choose your dungeon?

Comment (ming-tsai) [reply start-new-game.md]: Nice

**Comment by ming-tsai** [comment]
Hi @VJ1224, thanks for your contributions, and welcome to [MakeContributions](https://github.com/MakeContributions) 🎉

**Pull request merged**